File: 김강민/2023/Month_11/1102/42748.js
function solution(array, commands) {
    let result = [];
    let i = 0;
    let j = 0;
    let k = 0;

    for (let n = 0; n < commands.length; n++) {
        i = commands[n][0];
        j = commands[n][1];
        k = commands[n][2];

        let sliced = array.slice(i - 1, j);
        let sorted = sliced.sort((a, b) => a - b);

        result.push(sorted[k - 1]);
    }

    return result;
}

File: 김강민/2024/03/0307/42748.py
def solution(array, commands):
    answer = []
    arr = []
    for num in range(len(commands)):
        i = commands[num][0] -1
        j = commands[num][1]
        k = commands[num][2] -1
        
        arr = array[i:j]
        arr.sort()
        answer.append(arr[k])
        
    return answer